### DISCUSSION TITLE
Mehr als ein Victron Laderegler?

### DISCUSSION THREAD

**Discussion by sentinel1111** [skippermeister/OpenDTU-OnBattery#15] [General]
Hallo @skippermeister ,

Sollten mehr als ein Victron Laderegler mit der Software funktionieren?
Ich versuche nämlich zwei anzuschließen,bekomme es aber nicht hin.Einzel funktionieren beide.
Wenn aber beide in der Pinmapp.json eingetragen werden wird nur der erste aktiviert.
Der Eintrag ist so richtig?
"victron": {
            "rs232_rx": 40,
            "rs232_tx": 39,
	    "rs232_rx2": 42,
            "rs232_tx2": 41
}

**Comment by skippermeister** [comment]
Hi @sentinel1111,
Mit mehr als einem Victron habe ich das bislang nicht testen können.
Schick mir mal ein Log der Bootphase.
Der Eintrag im pinmapping sollte stimmen. Werden die Pins im Setup Menü Hardware korrekt angezeigt?

**Reply by sentinel1111** [reply]
Es wird nur ein Victron Laderegler als aktiv angezeigt.Benutzt wird die letzte Version von Github

![grafik](https://github.com/user-attachments/assets/1b80bdc7-37d6-4ccc-9736-3deae2cd1cf4)

Log von der Bootphase
[OdTUputty.txt](https://github.com/user-attachments/files/17508385/OdTUputty.txt)

**Reply by skippermeister** [reply]
@sentinel1111 
ich habe den Fehler gefunden. Falsches Index Variable im Pinmapping Parser für die Victron Pins verwendet. Danke für die Suche bei der Fehlersuche.
Ich habe eine neues Release hochgeladen und erstellt.
Desweiteren habe ich noch einen Parameter für über RS232 oder RS485 angeschlossenen Batterien im Batterie Setup Menü hinzugefügt. Es können jetzt die Baudraten 9600, 19200, 38400 und 115200 gewählt werden.
Und für JKBMS wurde im Batterie Setup Menü die Eingabe der "Empfohlenen Lade- und Entladeschlussspannung hinzugefügt. 

Funktionieren jetzt bei Dir beide Victrons ?
